accelerator/runtime_cli.c

```diff
@@ -0,0 +1,186 @@
+#include "runtime_cli.h"
+#include <stdbool.h>
+#include "common/logger.h"
+#include "connectivity/common.h"
+#include "connectivity/http/http.h"
+
+/* Forward declaration for state definition */
+static void do_help(char *);
+static void do_logger_switch(char *);
+static void do_logger_status(char *);
+static void do_redis_switch(char *);
+static void do_redis_memory(char *);
+static void handle_command(char *);
+static ta_cli_list *ta_state_ptr;
+static ta_cli_list redis_state, logger_state, start_state;
+
+/* Thread for runtime command line utility */
+void *cli_thread(void *arg) {
+  char command[128];
+  while (true) {
+    printf("Command> ");
+    fgets(command, 128, stdin);
+    command[strlen(command) - 1] = '\0';
+    if (strlen(command) == 0) continue;
+
+    /* Initialize the state then process the command */
+    ta_state_ptr = &start_state;
+    handle_command(command);
+  }
+  return NULL;
+}
+
+/* State definition */
+static ta_cli_list redis_state = {.size = 2,
+                                  {{.name = "switch",
+                                    .fn = do_redis_switch,
+                                    .type = CMD_LAST,
+                                    .describe = "Switch on/off redis support",
+                                    .example = "redis switch on/off"},
```

Comment:
Append `,` in the end of this statement. That is `.example = "redis switch on/off",`
So that, `clang-format` can indent in more elegant ways.